src/Elastic.Apm.AspNetFullFramework/AspNetHttpRequest.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System.Web;
+using Elastic.Apm.Config;
+using Elastic.Apm.Helpers;
+using Elastic.Apm.AspNetFullFramework.Extensions;
+using Elastic.Apm.AspNetCore;
```

Comment:
```suggestion
```

Once the namespace in [src/Elastic.Apm.AspNetFullFramework/AspNetHttpForm.cs](https://github.com/elastic/apm-agent-dotnet/pull/1806/files#diff-c4bb65fa7144201f6edfb58732e09879af82890cf77634cbc0b07cb44c58de65) is adapted - this can be removed as well.